package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A platform for electronic 1 on 1 philanthropy",
   "main": "index.js",
   "scripts": {
-    "start": "ts-node-dev src/app.ts"
+    "start": "ts-node-dev src/app.ts",
+    "watch": "webpack-dev-server --hot --inline --color --open"
```

Comment:
hey @PatrickMetsch this is awesome, will other developers know to automatically start calling watch? probably we should just update the `./hp localDev` to just call the new version...

but that would leave it so that server side changes don't auto refresh. which might be ok for now, since we are mostly developing on the browser code...

how about this: we have two ways to run the dev mode:

`./hp clientDev`
`./hp serverDev`

and then for backward compatibility (peeeps used to using localDev) we will just alias it to clientDev and output a message letting them know about the existence of the two.